### PR TITLE
Add specific CSS for print

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: assets/css/just-the-docs.scss [hunk 1]
```diff
@@ -42,4 +42,5 @@
 @import "./search";
 @import "./tables";
 @import "./code";
+@import "./print";
```
Comment (MattiSG) [inline assets/css/just-the-docs.scss]: This is pretty bad for perfs. Couldn't we load it in a separate file with `media=print`?

File: _sass/print.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
+@media print {
+  .page-content > :not(#markdown-toc) a:after,
+  .page-content > a:after {
+    content: " (" attr(href) ") ";
+    font-size: 0.8em;
+    font-weight: normal;
```
Comment (MattiSG) [inline _sass/print.scss]: Also `color: inherit` on `a`: there is no need to change color if it is not clickable.

File: _sass/print.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+@media print {
+  .page-content > :not(#markdown-toc) a:after,
+  .page-content > a:after {
+    content: " (" attr(href) ") ";
```
Comment (MattiSG) [inline _sass/print.scss]: > What about a monospace font to increase readability of URLs?

**Comment by Ndpnt** [comment]
@MattiSG everything is fixed

File: assets/css/print.css [hunk 4]
```diff
@@ -0,0 +1,63 @@
+@media print {
```
Comment (MattiSG) [inline assets/css/print.css]: > Unnecessary media query since the stylesheet is included at HTML level with `media="print"` 🙂

Comment (Ndpnt) [reply assets/css/print.css]: Argg… Yep, I forgot ro remove it…

**Pull request merged**